### PR TITLE
Add Example for Google Home using Thermostat

### PR DESCRIPTION
**Pull request by SamueldaCostaAraujoNunes**
I added an example for users who use Google Home, in order to control a Thermostat or an Air Conditioner!

### CODE REVIEW

File: arduino_examples/google_home_thermostat.ino [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+                turnOn(deviceId);
+            }
+            else
+            {
+                turnOff(deviceId);
+            }
+        }
+        else if (action == "action.devices.commands.ThermostatTemperatureSetpoint")
+        {
+            float value = json["value"]["acTemperatureSetpoint"];
```
Comment (kakopappa) [inline arduino_examples/google_home_thermostat.ino:107]: Is this correct?

Comment (SamueldaCostaAraujoNunes) [reply arduino_examples/google_home_thermostat.ino:107]: Yes, when there is a change in the temperature of the thermostats, I receive the command "action.devices.commands.ThermostatTemperatureSetpoint", which gives me a float corresponding to the desired temperature!

Comment (kakopappa) [reply arduino_examples/google_home_thermostat.ino:107]: I mean float value = json["value"]["acTemperatureSetpoint"]; <--- is this correct ?

Comment (SamueldaCostaAraujoNunes) [reply arduino_examples/google_home_thermostat.ino:107]: Ah ok. Yes, that's correct!

Comment (SamueldaCostaAraujoNunes) [reply arduino_examples/google_home_thermostat.ino]: I recompiled and tested it here, it worked correctly!

Comment (kakopappa) [reply arduino_examples/google_home_thermostat.ino]: Thanks

**Pull request merged**